find-stable-candidates

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+maint=${1}
+
+cd "$(git rev-parse --show-toplevel)"
+
+echo " Days │ category/package:SLOT"
+echo "──────┼────────────────────────────────────────────────────────────────────────"
+exec \
+	git grep -l "${maint}" '**/metadata.xml' \
```

Comment:
Thinking about it:

```suggestion
	git grep -l "${maint}" '*/*/metadata.xml' \
```

to avoid matching category's `metadata.xml`.

Feel free to update the other script as well or I'll do it later.

Reply:
Also, I think you can avoid backslashes by putting `|` at the end.

Reply:
Nice, thanks for both suggestions.